lib/TransportRPCClient.js

```diff
@@ -5,9 +5,11 @@ const uuid = require('uuid')
 const TransportRPC = require('./TransportRPC')
 
 class TransportRPCClient extends TransportRPC {
-  constructor (client, conf) {
+  constructor (client, conf, Socket) {
```

Comment:
to create an instance we need options specific to the implementation. right now we know the port very late in the bootstrap, as it is passed to the `.listen` method of the rpc-server:

https://github.com/robertkowalski/grenache-nodejs-ws/blob/33e9d82fd6c107073617fab767f3a9e633a44cca/lib/TransportRPCServer.js#L6-L9